### PR TITLE
fix: Deployment status devtron helm

### PR DESCRIPTION
**Pull request by shivani170**
# Description

deployment status card bug fix

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/app/details/appDetails/DeploymentStatusCard.tsx [hunk 1]
```diff
@@ -18,65 +25,94 @@ function DeploymentStatusCard({ deploymentStatusDetailsBreakdownData, loadingRes
...
+                    {deploymentStatusDetailsBreakdownData?.deploymentStatus === DEPLOYMENT_STATUS.INPROGRESS && (
                         <Timer className="icon-dim-16 mt-4" />
                     )}
                 </div>
-                <div className="fw-4 fs-12 cn-9 dc__ellipsis-right dc__mxw-inherit">
-                    by {deploymentStatusDetailsBreakdownData.triggeredBy || '-'}
-                </div>
+
+                {hideDeploymentStatusLeftInfo
+                    ? triggeredBy && triggeredBy
```
Comment (milindk8) [inline src/components/app/details/appDetails/DeploymentStatusCard.tsx:110]: why are using and operation between same variable

File: src/components/v2/appDetails/sourceInfo/environmentStatus/EnvironmentStatus.component.tsx [hunk 2]
```diff
@@ -153,7 +153,11 @@ function EnvironmentStatusComponent({
         return (
             appDetails?.lastDeployedTime && (
               <DeploymentStatusCard
+              isGitops={isGitops}
               deploymentStatusDetailsBreakdownData={deploymentStatusDetailsBreakdownData}
+              hideDeploymentStatusLeftInfo={!isGitops}
```
Comment (milindk8) [inline src/components/v2/appDetails/sourceInfo/environmentStatus/EnvironmentStatus.component.tsx:157]: can't we use isGitops directly as it is available on line 156

Comment (shivani170) [reply src/components/v2/appDetails/sourceInfo/environmentStatus/EnvironmentStatus.component.tsx:157]: Removed unnecessary one

File: src/components/app/details/appDetails/DeploymentStatusCard.tsx (hunk 1 above)
Comment (shivani170) [reply src/components/app/details/appDetails/DeploymentStatusCard.tsx]: fixed

**Pull request merged**